peps/pep-0738.rst

```diff
@@ -52,7 +52,7 @@ Android is broadly a POSIX platform, based on a Linux kernel and the
 ELF binary format. It does not use glibc, instead providing its own C
 library implementation called Bionic. As a result, it is generally not
 binary-compatible with any other Linux distribution, even if the architecture
-matches.
+matches. It also has a completely different filesystem layout.
```

Comment:
```suggestion
matches. It also has a filesystem layout that doesn't adhere to the Linux 
`Filesystem Hierarchy Standard (FHS) <https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard>`__.
```

Reply:
I've used a more general wording so it's clear it doesn't follow any Unix standard at all.